File: python/moving_zeros_to_the_end/kata.py
def move_zeros(array):
    """`codewars.com <https://www.codewars.com/kata/52597aa56021e91c93000cb0>`_"""
    out = []
    for e in array:
        if e is not 0:
            out.append(e)
    while len(out) < len(array):
        out.append(0)
    return out

File: go/count_the_monkeys/kata.go
package count_the_monkeys

// https://www.codewars.com/kata/56f69d9f9400f508fb000ba7

func monkeyCount(n int) []int {
	results := make([]int, n)
	for i := range results {
		results[i] = i + 1
	}
	return results
}
